Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/StubColumnRenderer.cs

```diff
@@ -31,24 +31,19 @@ public StubColumnRenderer (IResourceUrlFactory resourceUrlFactory)
     {
     }
 
-    protected override void RenderTitleCell (BocColumnRenderingContext<StubColumnDefinition> renderingContext, in BocTitleCellRenderArguments arguments)
+    public override void AddAttributesToRenderForTitleCell (BocRenderArguments bocTitleCellAttributeRenderingContext, in BocTitleCellRenderArguments arguments)
```

Comment:
We can probably simplify the variable name to `renderingContext` we did not add the fluff with `arguments` although the type is `BocTitleCellRenderArguments`